README.md

```diff
@@ -45,6 +45,14 @@ TAG=v0.4.0
 kubectl apply -f ./install/$TAG/kfserving.yaml --validate=false
 ```
 
+If you want to deploy the KFServing controller in `kubeflow` namespace.
+
+1. Set the namespace field to `kubeflow` in overlays/env/kustomization.yaml and [ingressGateway](https://github.com/kubeflow/kfserving/blob/master/config/configmap/inferenceservice.yaml#L94) to `kubeflow-gateway.kubeflow` in [config/configmap/inferenceservice.yaml](https://github.com/kubeflow/kfserving/blob/master/config/configmap/inferenceservice.yaml).
```

Comment:
Can we kustomize the gateway config also?

Reply:
Sure!